pom.xml

```diff
@@ -9,13 +9,20 @@
         <relativePath/> <!-- lookup parent from repository -->
     </parent>
     <groupId>mate.academy</groupId>
-    <artifactId>online-book-store01</artifactId>
+    <artifactId>book-shop</artifactId>
```

Comment:
That task requires such significant changes.
Please, make changes that are required by task only.